audformat/core/scheme.py

```diff
@@ -172,6 +172,21 @@ def __init__(
         self.maximum = maximum if self.is_numeric else None
         r"""Maximum value"""
 
+    @property
+    def is_numeric(self) -> bool:
+        r"""Data type is numeric.
+
+        Returns:
+            ``True`` if data type is numeric
+
+        """
+        return self.dtype in (define.DataType.INTEGER, define.DataType.FLOAT)
+
+    @property
+    def labels_as_list(self) -> typing.List:
+        r"""Scheme labels as list"""
```

Comment:
Maybe extend the docstring with  `Returns: ...` as we do with the other properties?

Reply:
I added the "Returns" section to the docstring.